modules/test/playwright/tests/journal-web/pages/JournalPage.ts

```diff
@@ -35,4 +40,83 @@ export class JournalPage {
 		await this.goto();
 		await this.templatesLink.click();
 	}
+
+	async assertJournalArticlePermissions(
+		title: string,
+		permissionLocators: string[]
+	) {
+		await this.goto();
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {
+				name: 'Permissions',
+			}),
+			trigger: this.page.getByLabel(`Actions for ${title}`, {
+				exact: true,
+			}),
+		});
+
+		for (const permissionsLocator of permissionLocators) {
+			await expect(
+				this.page
+					.frameLocator('iframe[title="Permissions"]')
+					.locator(permissionsLocator)
+			).toBeChecked();
+		}
+
+		await this.page
+			.frameLocator('iframe[title="Permissions"]')
+			.getByRole('button', {name: 'Cancel'})
+			.click();
+	}
+
+	async deleteJournalArticle(title: string) {
+		await this.goto();
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {
+				name: 'Delete',
+			}),
+			trigger: this.page.getByLabel(`Actions for ${title}`, {
+				exact: true,
+			}),
+		});
+	}
+
+	async setJournalArticlePermissions(
+		articles: Locator[],
+		permissionLocators: string[]
+	) {
+		await this.goto();
+
+		for (const article of articles) {
+			await article.getByTitle('Select').check();
+		}
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {
+				name: 'Permissions',
+			}),
+			trigger: this.page.getByTitle('Actions', {exact: true}),
+		});
+
+		for (const permissionsLocator of permissionLocators) {
+			await this.page
+				.frameLocator('iframe[title="Permissions"]')
```

Comment:
Maybe we can move the `.frameLocator('iframe[title="Permissions"]')` to JournalPage page member, like `PermissionsFrame` or something.

Reply:
I will also change this, thanks!